support-frontend/assets/helpers/theGrid.js

```diff
@@ -108,7 +108,8 @@ export const imageCatalogue: {
   printFeaturePackshot: '017a2f5c27394635b53c414962bbb775ce9b131d/5_39_1572_861',
   printCampaign2020: 'e50943f03b58caf6fde78d81eb65c92c048701c9/1844_0_1674_1006',
   printCampaignHero: 'fff86e98dbe83b36892baf7942f30fd3bcbcaee6/286_215_714_285',
-  printCampaigniMovo: 'f257e33b922f09ca180cc9205e0c766c2c2d529c/28_0_694_694',
+  printCampaigniMovo: '3cca8e05f3253f5a14329429ec3f6f139f84929a/0_21_562_337',
```

Comment:
I think `printCampaignDigitalVoucher` would be a better name, we don't generally use iMovo in code I don't think